apps/ejabberd/src/mod_caps.erl

```diff
@@ -575,22 +575,22 @@ concat_info(Els) ->
                              Els)).
 
 concat_xdata_fields(Fields) ->
```

Comment:
This function isn't difficult to read in my opinion and I don't think it's unclean. I've decided to only improve the style a bit, e.g. use tuple in accumulator instead of list and replace `xml` with `exml`.